tests/test_pythia8_fill.py

```diff
@@ -0,0 +1,100 @@
+import chromo
+from chromo.common import EventData
+from .util import run_in_separate_process
+import pytest
+import sys
+
+pytestmark = pytest.mark.skipif(
```

Comment:
Do you need the return value? If yes, call it `_` because its name is irrelevant and the `_` signifies that. Note that there is also `pytest.skipif`, maybe you want that instead?

Reply:
Removed. No, return value is not needed. `pytest.skipif` doesn't exist, only `pytest.mark.skipif`.

Reply:
It turns out that the name is required https://docs.pytest.org/en/7.1.x/reference/reference.html#globalvar-pytestmark

Reply:
This code does not require the variable:
```py
if sys.platform == "win32":
     pytest.skip("Pythia8 does not run on windows", allow_module_level=True)
```

Reply:
You are right, `pytest.skipif` does not exist, but `pytest.skip` does. When you use `skip` as a decorator, the `if` has to be part of the function, but when you call `skip` in normal imperative code, you can also write the `if` in normal Python.

Reply:
I see. Thanks for explanation.